Framework/Core/include/Framework/StepTHn.h

```diff
@@ -87,6 +91,7 @@ class StepTHnT : public StepTHn
  public:
   StepTHnT() : StepTHn() {}
   StepTHnT(const Char_t* name, const Char_t* title, const Int_t nSteps, const Int_t nAxis, Int_t* nBins, std::vector<Double_t> binLimits[], const char** axisTitles);
+  StepTHnT(const char* name, const char* title, const int nAxes, const int* nBins, const double* xmin, const double* xmax, const int nSteps);
```

Comment:
The second constructor is for equidistant binning only?
The std::vector is used to allow both, equidistant (2 entries min/max), and custom bin edges.
The different order of the arguments in both constructors is not ideal I think.

Reply:
Yes, the second ctor is for equidistant binning only, but the registry will then adjust the binning afterwards in the same manner as it is done for THn{Sparse} histograms.
Regarding the order of the arguments, I agree that this is not good. I will move nSteps between title and nAxes.